src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -105,16 +120,21 @@ internal async override Task<VSReferenceItem[]> HandleRequestAsync(ReferencePara
                 PartialResultToken = token // request.PartialResultToken
             };
 
+            _logger.LogInformation("Attaching to progress listener.");
```

Comment:
I feel like this should be in the progress listener; however, the progress listener exists in the contained language framework. I wonder if we should be changing the model for utilizing some of these contained language types so we can be integrating logs into them... Hmmm. @ToddGrun @jimmylewis @alexgav thoughts?